doc/source/user_guide/data_types.rst

```diff
@@ -124,6 +124,42 @@ may write::
    >>> plt.imshow(out)
 
 
+Working with OpenCV
+===================
+
+It is possible that you may need to use an image created using ``skimage`` with
+OpenCV or vice versa. An image in OpenCV is also a Numpy array, however its
```

Comment:
Also, I think this statement is a bit misleading: OpenCV images are not numpy arrays.  But you can say something like "OpenCV image data can be accessed (without copying) in NumPy.  We should keep two things in mind: OpenCV uses BGR (instead of scikit-image's RGB) for color images, and a data-type of uint8 by default (see ...).